src/components/Table/components/EditColumnsButton.tsx

```diff
@@ -43,28 +43,30 @@ export function EditColumnsButton<R extends Kinded>(props: EditColumnsButtonProp
       : trigger.name,
   );
 
-  const { options } = useMemo(() => {
-    return columns.reduce(
```

Comment:
This reduce was necessary when it was written, b/c two collections were being created at the same time, but the 2nd collection was removed in an earlier PR, so now it can be just a filter.